content/en/docs/how-tos/use-registries-in-build-farm.md

```diff
@@ -35,20 +35,22 @@ registry is [registry.ci.openshift.org](https://registry.ci.openshift.org).
 
 ## How do I log in to pull images that require authentication?
 
-All registries are the internal OpenShift image registry for the cluster they reside on, so authenticating to the registry
-requires authentication to the cluster that hosts it. Once logged in to the OpenShift cluster, the `oc` CLI can be used to
-authenticate to the registry in question. For example, for the [registry.build01.ci.openshift.org](https://registry.build01.ci.openshift.org)
-registry, the cluster is `build01`. Using the [the list of clusters](/docs/getting-started/useful-links/#clusters),
-the console URL for this cluster is found to be [console.build01.ci.openshift.org](https://console.build01.ci.openshift.org).
+If you want to e.g. download pre-release images from [the release page](https://openshift-release.apps.ci.l2s4.p1.openshiftapps.com/),
+then you will need to authenticate.
```

Comment:
1) Maybe let's just add this as a new, targeted question? "How do I dowload release images from release page" and instantiate the answer as "auth against app.ci cluster as described in in [link here]".

2) Pleas e use the https://amd64.ocp.releases.ci.openshift.org/ URL for the release-ctonrroller frontend

Reply:
> Pleas e use the https://amd64.ocp.releases.ci.openshift.org/ URL for the release-ctonrroller frontend

Ah nice!  I hadn't seen that we again have nice readable alias for the release controller.

Actually, the real fix may be to just add this information into that web page.

Reply:
https://github.com/openshift/release-controller/pull/258